### PR TITLE
ISPN-8693 Improve ControlledRpcManager and BlockingLocalTopologyManager

### PR DESCRIPTION
**Pull request by danberindei**
Fix some tests broken by the initial pull request

https://issues.jboss.org/browse/ISPN-8693

### CODE REVIEW

**Comment by gustavonalle** [comment]
Failures fixed, do you want to take a look before integrating @pruivo or @rvansa ?

File: core/src/test/java/org/infinispan/util/BlockingLocalTopologyManager.java [hunk 1]
```diff
@@ -38,6 +40,7 @@
    private final Address address;
    private final String expectedCacheName;
    private final BlockingQueue<Event> queuedTopologies = new LinkedBlockingQueue<>();
+   private final List<Event> skippedEvents = new ArrayList<>();
```
Comment (pruivo) [inline core/src/test/java/org/infinispan/util/BlockingLocalTopologyManager.java]: unused

**Comment by pruivo** [comment]
@gustavonalle just a minor comment. it looks good to me

**Comment by gustavonalle** [comment]
Merged, thanks @danberindei !

**Pull request merged**